src/popover/popover.ts

```diff
@@ -0,0 +1,16 @@
+import {Component, Input, ChangeDetectionStrategy} from '@angular/core';
+
+@Component({
+  selector: 'ngb-popover-window',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {'[class]': '"popover " + placement', 'role': 'tooltip'},
```

Comment:
Oh right, I meant that if you do `<ngb-popover-window class="foo bar baz"...` those 3 classes would be removed to add the popover-placement one.